### ISSUE TITLE
Update checkout object's customAttributes using mutations - Express JS

### ISSUE THREAD

**Issue by abhilash29** [Shopify/js-buy-sdk#498]
I'm currently using Express JS. I've successfully integrated the JS buy SDK and I'm successfully able to create/update/delete checkouts and their lineItems.
I also managed to create customAttributes during checkout creation.
However, what I'm not able to figure out is the correct syntax/code to use mutations as mentioned in [checkoutAttributesUpdate][1] link .
Unfortunately, there doesn't seem to be an out of the box function to do this .

This is how I added customAttributes.

   
```
const input = {
    customAttributes: customAttributes
};
client.checkout.create(input).then((checkout) => {
    //Remaining stuff
  
});

```
  [1]: https://help.shopify.com/api/storefront-api/reference/mutation/checkoutattributesupdate#interactive-example

**Comment by AmeliaBR** [comment]
I'm having the same issue, and this seems like a significant gap in the API to me.

As far as I can tell, any time I need to update cart attributes, I either need to 

- create a new checkout with a copy of all existing data (e.g. line items) about the cart.
- process the GraphQL mutation myself

An `update` method in the [`CheckoutResource` class](https://github.com/Shopify/js-buy-sdk/blob/master/src/checkout-resource.js]) would be very useful.

**Comment by thohot303** [comment]
Unbelievable that Shopify just updated the Buy Button implementation without having proper replacement methods in place. I can now no longer update custom attributes via permalinks and an update method on the CheckoutResource object is not available.

**Comment by StefanSlehta** [comment]
You can now use the [update](https://github.com/Shopify/js-buy-sdk/blob/master/src/checkout-resource.js#L89) method  in version `1.5.0` to add manipulate `customAttributes` on a `Checkout`

**Comment by thohot303** [comment]
Thanks, much appreciated. When will this be moved to the current implementation, though?

**Comment by StefanSlehta** [comment]
The only thing that's been changed in `1.6.0` was the addition of this method, so you can update to the newest version without breaking any functionality.

**Comment by thohot303** [comment]
My app requires users to install & modify the Shopify-generated Buy Button code.
The [script URL](https://sdks.shopifycdn.com/buy-button/latest/buy-button-storefront.min.js) points to version **1.4.0**.

**Comment by StefanSlehta** [comment]
It's published now, so the `1.6.0` should be loading the

**Comment by thohot303** [comment]
Awesome, works like a charm now, thanks!
